addons/website_event_exhibitor/views/event_exhibitor_templates_page.xml

```diff
@@ -152,6 +157,10 @@
                  placeholder="e.g. &quot;Openwood specializes in home decoration...&quot;"/>
             <div class="alert alert-info mx-3 mt-3 o_wesponsor_exhibitor_main_empty_website_descr_warning">
                 The sponsor website description is missing.
+                <a class="alert-link" t-attf-href="?{{ keep_query('*', enable_editor=1) }}">
+                    <i class="oi oi-fw oi-arrow-right"></i>
+                    Write one.
+                </a>
```

Comment:
Same here, wouldn't add this
You can just edit it in-place, it's not really that much more convenient.